src/services/formatting/formattingScanner.ts

```diff
@@ -278,5 +280,24 @@ namespace ts.formatting {
             }
             return tokenInfo;
         }
+
+        function skipToEndOf(node: Node): void {
+            scanner.setTextPos(backUpWhitespace());

```

Comment:
usually `node.end` points to something that is not whitespace. Can you please give an example of a case when end of node points to a whitespace so we need to step back to the first non-whitespace position?